### DISCUSSION TITLE
Scrolling of the list of views

### DISCUSSION THREAD

**Discussion by epzt** [senshu/Sozi#569] [General]
Dear all,
I have a presentation with more than 30 slides/views. My problem comes from the fact that apparently, there is no scroll capabilities of the view list (at the bottom of Sozi), so that views after, in my case, number 21, is not visible as it "outputs" from the viewport of the window.
Is there a way to turn around this problem ?
Regards

**Comment by aumouvantsillage** [comment]
There should be a horizontal scroll bar at the bottom of the window.
Can you, please, open an issue and provide more details about the context (operating system, version of Sozi, ...)
Thanks.

**Reply by epzt** [reply]
Thanks for your advise.
I have opened an issue now.
Regards
